.circleci/config.yml

```diff
@@ -293,7 +293,7 @@ commands:
     steps:
       - run:
           name: Linting
-          command: ./.circleci/lint-packages.sh
+          command: yarn workspaces foreach -piv  --exclude=functional-tests  --exclude=fxa-payments-server  run lint
```

Comment:
Nit, I think there's a double space between `-piv` and `--exclude`, and after `payments-server` and `run lint`.